hidden.php

```diff
@@ -0,0 +1,36 @@
+<?php 
+require_once 'functions.php';
+
+$db = connectDatabase();
+$pMin = 1;
+$pMax = 50000;
+$gMin = 1;
+$gMax = 5000;
+$rMin = 1;
+$rMax = 12;
+if (
+  isset($_POST['name']) && ($_POST['name'] !== '') &&
```

Comment:
you can use a function called `empty()` instead here which will do both checks in 1 go